include/cantera/oneD/StFlow.h

```diff
@@ -242,6 +242,12 @@ class StFlow : public Domain1D
         m_kin->getNetProductionRates(&m_wdot(0,j));
     }
 
+    virtual void updateProperties(size_t jg, double* x, double* rsd, int* diag,
+                                  double rdt, size_t jmin, size_t jmax);
+
+    virtual void evalResidual(double* x, double* rsd, int* diag,
+                              double rdt, size_t jmin, size_t jmax);
```

Comment:
This function needs a docstring, specifically to explain the relationship between this method and `eval`.